src/components/Buttons.jsx

```diff
@@ -6,17 +6,17 @@ const Buttons = ({ buttons, onButtonPress }) => {
       className="grid grid-cols-[repeat(4,1fr)] gap-[0.3rem] text-xl 
```

Comment:
For this grid-cols, you can use grid-cols-4, which is equivalent to the one you are using.